docs/src/app/components/pages/components/Stepper/VerticalLinearStepper.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+  Stepper,
+  VerticalStep as Step,
+} from 'material-ui/lib/stepper';
+
+import {
+  Paper,
+  FontIcon,
+  RaisedButton,
+  FlatButton,
+} from 'material-ui';
+
+import getMuiTheme from 'material-ui/lib/styles/getMuiTheme';
+import themeDecorator from 'material-ui/lib/styles/theme-decorator';
+import colors from 'material-ui/lib/styles/colors';
+
+const muitheme = getMuiTheme({
+  palette: {
+    primaryColor: colors.teal500,
```

Comment:
=D Because i love that color ; but anyway i will remove it. Thanks for your feedbacks